flocker/node/script.py

```diff
@@ -198,7 +198,7 @@ def get_service(self, reactor, options):
         ip = _get_external_ip(host, port)
         return AgentLoopService(
             reactor=reactor,
-            # Temporary hack, to be fixed in FLOC-1733 probably:
+            # Temporary hack, to be fixed in FLOC-1783 probably:
             deployer=self.deployer_factory(node_uuid=ip_to_uuid(ip),
```

Comment:
`IDeployer` needs to know the UUID of the node that it's running on....but that bit has already been done.